### ISSUE TITLE
My Site: VoiceOver: the header on the blog main screen should be one accessible element

### ISSUE THREAD

**Issue by designsimply** [wordpress-mobile/WordPress-iOS#12180]
Steps to reproduce:

1. Go to Settings > General > Accessibility > VoiceOver and toggle the setting on.
1. Go to My Sties and select a site.
1. Interact with the header.

**Result:** the whole header should be one accessible element.

<img src="https://user-images.githubusercontent.com/1119271/61653190-f222dc00-ac76-11e9-83c7-07cd90b47257.PNG" width="270" alt="IMG_2914">
<sup>Tested with WPiOS 12.8.1 on iPhone 6S iOS 12.3.1.</sup>

<sub>(internal reference: p77Llu-bQ6-p2 h/t @etoledom)</sub>
